infrastructure/resource_group_level/modules/function-acbs.bicep

```diff
@@ -10,24 +10,15 @@ param nodeDeveloperMode bool
 
 param resourceNameFragment string = 'function-acbs'
 
+param settings object
```

Comment:
Overall, we generally have the following classes of settings:
`settings`: non-sensitive values from `vars` or just hard coded in the main file, observed in the CLI scripts.
`secureSettings`: sensitive values from GH secrets, observed in the CLI scripts.
`additionalSecureSettings`: values that appeared when I performed an export from the `dev` environment, that aren't in the CLI scripts. Perhaps some can be removed.
`staticSettings`: settings that are either hard-coded or can be calculated inline.
`secureConnectionStrings`: connection strings that come from secrets and are in the CLI scripts. As discussed, we may want to move these to normal secureSettings at some point.
`additionalSecureConnectionStrings`: connection strings that appeared when I performed an export on `dev`.